apps/labs/posts/code-generator-and-nebari.md

```diff
@@ -0,0 +1,159 @@
+---
+title: "Bridging Data Science Tools with PyTorch-Ignite's Code Generator and Nebari"
+published: September 20, 2023
+authors: [aryan-gupta]
+description: 'A summary of my contributions to the Code-Generator Project and PyTorch-Ignite ecosystem in the past few months as Quansight Labs intern and my learnings in the process.'
+category: [PyData ecosystem, Machine Learning, OSS Experience]
+featuredImage:
+  src: /posts/code-generator-and-nebari/hero-feature-image.png
+  alt: 'An illustration of Code-Generator app integration with Nebari and other ML based libraries.'
+hero:
+  imageSrc: /posts/code-generator-and-nebari/feature_and_hero.svg
+  imageAlt: 'An illustration of Code-Generator app integration with Nebari and other ML based libraries.'
+---
+
+To the readers, I am Aryan Gupta(@[guptaaryan16](https://github.com/guptaaryan16/)), an EE Junior at IIT Roorkee, and this summer, I had a chance to work on PyTorch-Ignite’s Code-Generator Project, a tailor-made web application to help machine learning researchers and enthusiasts and also keeping in mind the growing Kaggle community.
+
+# The Project Itself
+
+Let’s see the project itself. [Code-generator](https://code-generator.pytorch-ignite.ai/) is a [Vue.js](https://vuejs.org/) application that streamlines the process of working on machine learning tasks. The app generates preconfigured code templates for tasks like vision classification, text classification, and other common themes in ML competitions.
+
+The aim is not to be another abstraction over PyTorch and PyTorch-Ignite. Instead, it is a web app that generates the boilerplate code, which means you still have complete control over the generated code. You can start by selecting the minimal template like ‘Text-Classification’ and turn options on or off according to your needs, like a particular logger like [Tensorboard](https://tensorboard.dev/) or an arg-parser like [Python-Fire](https://github.com/google/python-fire/blob/master/docs/guide.md). This app can be helpful in data science competitions on Kaggle or in writing papers, especially with easy integration of loggers and checkpoint handlers. Also, the templates are well-tested on CI and hopefully will require minimal changes to work on your workflows.
```

Comment:
To make it clearer, I suggest
```suggestion
The aim is not to be another abstraction over PyTorch and PyTorch-Ignite. Instead, Code-Generator is a web app that generates the boilerplate code, which means you still have complete control over the generated code. You can start by selecting the minimal template like ‘Text-Classification’ and turn options on or off according to your needs, select a particular logger like [Tensorboard](https://tensorboard.dev/) or an arg-parser like [Python-Fire](https://github.com/google/python-fire/blob/master/docs/guide.md). This app can be helpful in data science competitions on Kaggle or in writing papers, especially with easy integration of loggers and checkpoint handlers. Also, the templates are well-tested on CI and hopefully will require minimal changes to work on your workflows.
```

(Also, tiny detail: looking at the website, I see mentions of Code-Generator, Code Generator and here in your blog post I also see Code-generator. Is the capitalization/hyphenization something we want to make consistent?)